index.html

```diff
@@ -26,7 +32,158 @@ <h1>I’m Mariane Glad to see you!</h1>
+            <a class=img href="#"><img src="images/portfolio- snapshot-2.svg" alt="Portfolio-Snapshot-2"></a>
+
+            <h1 class="project-title">Multi-Post Stories</h1>
+            <ul class="project-name">
+                <li class="sub-title"><a href="#">CANOPY</a></li>
+                <li class="bullet">&#x2022;</li>
+                <li><a class="backend-yr" href="#">Back End Dev</a></li>
+                <li class="bullet">&#x2022;</li>
+                <li><a class="backend-yr" href="#">2015</a></li>
+            </ul>
+            <p class="description">A daily section of privately personalized reads; no accounts or sign-ups
+                required.
+            </p>
+
+            <ul class="classification">
+                <li class="lang"><a href="#">html</a></li>
+                <li class="lang"><a href="#">css</a></li>
+                <li class="lang"><a href="#">javaScript</a></li>
+            </ul>
+            <div>
+                <button class="btn" type="button" class="default_btn">See Project</button>
+            </div>
+        </div>
 
+        <!--Portfolio Card -3-->
+        <div class="portfolio-snapshot">
+            <a class=img href="#"><img src="images/portfolio- snapshoot-3.svg" alt="Portfolio-Snapshot-3"></a>
+
+            <h1 class="project-title">Tonic</h1>
+            <ul class="project-name">
+                <li class="sub-title"><a href="#">CANOPY</a></li>
+                <li class="bullet">&#x2022;</li>
+                <li><a class="backend-yr" href="#">Back End Dev</a></li>
+                <li class="bullet">&#x2022;</li>
+                <li><a class="backend-yr" href="#">2015</a></li>
+            </ul>
+            <p class="description">A daily section of privately personalized reads; no accounts or sign-ups
+                required.
+            </p>
+
+            <ul class="classification">
+                <li class="lang"><a href="#">html</a></li>
+                <li class="lang"><a href="#">css</a></li>
+                <li class="lang"><a href="#">javaScript</a></li>
+            </ul>
+            <div>
+                <button class="btn" type="button" class="default_btn">See Project</button>
+            </div>
+        </div>
+
+        <!--Portfolio Card -4-->
+        <div class="portfolio-snapshot">
+            <a class=img href="#"><img src="images/portfolio-snapshoot-4.svg" alt="Portfolio-Snapshot-4"></a>
+
+            <h1 class="project-title">Multi-Post Stories</h1>
+            <ul class="project-name">
+                <li class="sub-title"><a href="#">CANOPY</a></li>
+                <li class="bullet">&#x2022;</li>
+                <li><a class="backend-yr" href="#">Back End Dev</a></li>
+                <li class="bullet">&#x2022;</li>
+                <li><a class="backend-yr" href="#">2015</a></li>
+            </ul>
+            <p class="description">A daily section of privately personalized reads; no accounts or sign-ups
```

Comment:
- [ ] I think it's better to avoid lines of code longer than 100 characters following the best practices of coding. For more information [click here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Reply:
Thank for the information. Since this is not a project requirement, I will take this as optional. Thank you.